apps/frontend/src/app/Data/Characters/Baizhu/index.tsx

```diff
@@ -361,24 +361,18 @@ const sheet: ICharacterSheet = {
             unit: 's',
             fixed: 1,
           },
-          {
-            text: ct.chg('burst.skillParams.2'),
-            value: dm.burst.shieldInterval,
```

Comment:
Shouldn't we still display shield interval?

Reply:
![image](https://user-images.githubusercontent.com/121315943/236055131-eed9d55e-45db-454b-b39d-bdb879274ec4.png)